From aed87c902592654539657da8592f55194b8070d1 Mon Sep 17 00:00:00 2001
From: Daniel Hsu <Daniel-Hsu@quantatw.com>
Date: Fri, 24 Jul 2026 15:25:19 +0800
Subject: [PATCH] configurations: revise 200G BRCM NIC configuration

Add an MCTPI2CTarget entry to allow MCTPReactor to automatically
assign an EID to the BRCM NIC over I2C, removing the need for manual
EID configuration.

Update the BRCM NIC temperature sensor naming.

Update thresholds

Tested on Yv5:
- EID is correctly assigned
- Version object is properly exposed
- Temperature sensor is exposed as NIC0_TEMP_C
---
 .../broadcomm/200g_1p_ocp_mezz.json           | 28 ++++++++++++-------
 1 file changed, 18 insertions(+), 10 deletions(-)

diff --git a/configurations/broadcomm/200g_1p_ocp_mezz.json b/configurations/broadcomm/200g_1p_ocp_mezz.json
index 3f028b7..a9c29b6 100644
--- a/configurations/broadcomm/200g_1p_ocp_mezz.json
+++ b/configurations/broadcomm/200g_1p_ocp_mezz.json
@@ -3,37 +3,45 @@
         {
             "Address": "$address",
             "Bus": "$bus",
-            "Name": "BRCM OCP NIC FRU $bus",
+            "Name": "BRCM OCP NIC FRU $bus / 10",
             "Type": "EEPROM"
         },
+        {
+            "Address": "0x32",
+            "Bus": "$bus",
+            "Name": "NIC",
+            "Type": "MCTPI2CTarget"
+        },
         {
             "Address": "0x1f",
             "Bus": "$bus",
             "Labels": [
                 "temp2"
             ],
-            "Name": "NIC Temp $bus",
-            "Name1": "NIC Temp $bus",
+            "Name": "NIC$bus / 10 TEMP_C",
+            "Name1": "NIC$bus / 10 TEMP_C",
             "Thresholds": [
                 {
-                    "Direction": "less than",
+                    "Direction": "greater than",
+                    "Hysteresis": 2.1,
                     "Index": 2,
-                    "Name": "lower critical",
+                    "Name": "upper critical",
                     "Severity": 1,
-                    "Value": 5
+                    "Value": 105
                 },
                 {
                     "Direction": "greater than",
+                    "Hysteresis": 2.3,
                     "Index": 2,
-                    "Name": "upper critical",
-                    "Severity": 1,
-                    "Value": 105
+                    "Name": "upper non recoverable",
+                    "Severity": 4,
+                    "Value": 115
                 }
             ],
             "Type": "TMP421"
         }
     ],
-    "Name": "BRCM OCP NIC FRU $bus",
+    "Name": "BRCM OCP NIC $bus / 10",
     "Probe": "xyz.openbmc_project.FruDevice({'BOARD_PRODUCT_NAME': 'BRCM 200G 1P.*57508 OCP Mezz'})",
     "Type": "Board",
     "xyz.openbmc_project.Inventory.Decorator.Asset": {
-- 
2.53.0-Meta

